File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        float result = 0;
        
        if(price >= 500000) {
            result = price * 0.8f;
        } else if(price >= 300000) {
            result = price * 0.9f;
        } else if(price >= 100000) {
            result = price * 0.95f;
        } else {
            result = price;
        }
        return (int) result;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    answer = 0
    
    if 500000 <= price:
        answer = int(price * 0.8)
    elif 300000 <= price:
        answer = int(price * 0.9)
    elif 100000 <= price:
        answer = int(price * 0.95)
    else:
        answer = price
    
    return answer